gcn/notices/icecube/GoldAndBronze.schema.json

```diff
@@ -2,11 +2,11 @@
   "$id": "https://gcn.nasa.gov/schema/gcn/notices/icecube/GoldAndBronze.schema.json",
   "$schema": "https://json-schema.org/draft/2020-12/schema",
   "type": "object",
-  "title": "GoldAndBronze",
-  "description": "GoldAndBronze events",
+  "title": "IceCubeGoldAndBronzeTracks",
+  "description": "IceCube Astrotrack Gold And Bronze track alert events",
   "properties": {
     "type": {
```

Comment:
We've recently updated the schema to a more human-readable format that uses fewer curly braces {}. Please update your schema by replacing 'properties' with 'allOf' (ex:https://github.com/nasa-gcn/gcn-schema/blob/main/gcn/notices/fermi/lat/Alert.schema.json).

Reply:
Thanks for the suggestion.  I'm far from a JSON schema expert, but doesn't allOf require that all items are provided all the time?  I think that will work here, as we don't have any items that will only appear conditionally.    

We're working on other schema, namely for LVK followup where some items will be conditionally included (say if there'a a neutrino in spatial and temporal coincidence with the LVK localization.  In these cases, allOf won't work as a replacement to properties and required?  Is there a preferred way to handle these?  thanks

Reply:
Additionally:  for the IceCube specific items in the schema, they should be locally defined in the schema file to include with allOf, defined in a separate file or look to make additions to the core area?

Reply:
Yes, it is true that allOf requires all the schemas referred to. However, if you need more flexibility, for instance, to accommodate LVK localization, you can use anyOf (https://json-schema.org/understanding-json-schema/reference/combining.html#anyof) instead. We would suggest to use combination of allOf (for must schema sets) and anyOf (for optional schemas). I am also trying the JSON for the first time, thanks to @dakota002!

Regarding your additional question, we also had a discussion about it, we decided to define the Specific schema fields within the same schema instead of referring to a separate file. Thus, please go with defining the IceCube specific schema within alert schema, as already done, flexible structure would be: "allOf": [{$ref1}, {$ref2}...], "anyOf": [{$ref1}, {$ref2}...], "properties":{specific schema}.